lib/checks/shared/has-alt.js

```diff
@@ -1,4 +1,6 @@
-var nn = node.nodeName.toLowerCase();
-return (
-	node.hasAttribute('alt') && (nn === 'img' || nn === 'input' || nn === 'area')
-);
+const { nodeName } = virtualNode.props;
+if (['img', 'input', 'area'].includes(nodeName) === false) {
```

Comment:
Why not `!['img', 'input', 'area'].includes(nodeName)`?

Reply:
Easier to spot the negation this way. Doesn't really matter I guess.